### PR TITLE
Add vast2nix

### PR DESCRIPTION
**Pull request by GTrunSec**
A highly declarable, flexible framework for the data science field. powered by nix, Some functions or attributions are specially designed for users of Tenzir.

- https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/tenzir

### CODE REVIEW

**Comment by GTrunSec** [comment]
The data-science-threat-intelligence should be a part of [vast2nix](https://github.com/GTrunSec/vast2nix).

**Review by dit7ya — COMMENTED** [review]
I think it makes sense to include a clear purpose of what the project does _on top of VAST_, and a getting started guide showcasing the features that vanilla VAST does not provide.

**Comment by GTrunSec** [comment]
Basically, the vast2nix is ​​a nix-based devops framework and all-encompassing solution, using the same working methods and theories as [cardano-world](https://github.com/input-output-hk/cardano-world#cardano-world-repository), and has the same purpose. Each cell of vast2nix plays a different role, and these roles undertake tasks associated with it. Since this concept requires long-term maintenance and has a complex picture in the maintainer's mind, I will strip the quarto and data-science parts to a minimum interface and provide a user guide.

In fact, it has worked if it does not involve an override/overlay mechanism for complex parameters/attributes.

- For example, the OCI solution for the quarto
```bash
# copy the image to your docker directly
nix run github:gtrunsec/vast2nix#x86_64-linux.quarto.containers.dev.copyToDockerDaemon
# or copy the image to a docker-achieve format
nix run github:gtrunsec/vast2nix#x86_64-linux.quarto.containers.dev.copyTo 'docker-archive://$PWD/image'
# load this image manually 
docker load -i $PWD/image
# or copy the image to your remote registry
nix run github:gtrunsec/vast2nix#x86_64-linux.quarto.containers.dev.copyToRegistry docker://registry.example.com/quarto-dev:latest         
```
  - How do I extend the packages for the quarto
     -  modify here. https://github.com/GTrunSec/vast2nix/blob/main/nix/quarto/packages.nix#L12
- what is the minimal interface for the users? 
  - https://github.com/GTrunSec/data-science-threat-intelligence/blob/main/users/tenzir/flake.nix#L19. No complex options and mechanisms; just add a new package, then go.

- Enter the devshell environment in docker
```
docker run -it -v "$(pwd):/work" --entrypoint /bin/bash quarto-dev`
``

**Comment by GTrunSec** [comment]
If I want to have a data-science/jupyterlab environment,  what kind of solution is it?

No big changes from before. Just rename the `quarto ` to `data-science`
```
# copy the image to your docker directly
nix run github:gtrunsec/vast2nix#x86_64-linux.data-science.containers.dev.copyToDockerDaemon
# or copy the image to a docker-achieve format
nix run github:gtrunsec/vast2nix#x86_64-linux.data-science.containers.dev.copyTo 'docker-archive://$PWD/image'
# or copy the image to your remote registry
nix run github:gtrunsec/vast2nix#x86_64-linux.data-science.containers.dev.copyToRegistry docker://registry.example.com/quarto-dev:latest         
```

File: readme.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ A curated list of awesome things related to <a href='https://vast.io/'>VAST</a>
 
 ## Community Projects
 
+- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) A Nix-based DevOps framework for the definition of declarative and reproducible environments jobs and. 
+  - check all available tasks:
+```bash
+git clone https://github.com/GTrunSec/vast2nix && cd vast2nix
```
Comment (dit7ya) [inline readme.md]: I think the getting started guide should be part of the README of your project.

**Comment by GTrunSec** [comment]
I have added a simple readme in vast2nix. and migrated the `DockerFile` to nix-based DockerImage. 
- also applies to podman: https://github.com/GTrunSec/vast2nix/blob/main/nix/vast/oci-images/default.nix#L16
```
docker pull ghcr.io/gtrunsec/vast:latest
docker run --user vast -it --rm -p 42000:42000 \
  -v $(pwd)/test ghcr.io/gtrunsec/vast:latest start
```

File: readme.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ A curated list of awesome things related to <a href='https://vast.io/'>VAST</a>
 
 ## Community Projects
 
+- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) A Nix-based DevOps framework for the definition of declarative and reproducible environments jobs and.
```
Comment (dit7ya) [inline readme.md]:
```suggestion
- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs. 
```

File: readme.md [hunk 3]
```diff
@@ -34,6 +34,13 @@ A curated list of awesome things related to <a href='https://vast.io/'>VAST</a>
 
 ## Community Projects
 
+- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) A Nix-based DevOps framework for the definition of declarative and reproducible environments jobs and. 
+  - check all available tasks:
+```bash
+git clone https://github.com/GTrunSec/vast2nix && cd vast2nix
+nix develop --command std list
+```
+
```
Comment (dit7ya) [inline readme.md]:
```suggestion

```
    
```

**Review by dit7ya — CHANGES_REQUESTED** [review]

File: readme.md [hunk 4]
```diff
@@ -34,6 +34,7 @@ A curated list of awesome things related to <a href='https://vast.io/'>VAST</a>
 
 ## Community Projects
 
+- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs.
```
Comment (dit7ya) [inline readme.md]:
```suggestion
- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs. 
```

**Review by dit7ya — CHANGES_REQUESTED** [review]
One last change I missed. Can you please squash all commits and force-push?

File: readme.md (hunk 4 above)
Comment (GTrunSec) [reply readme.md]: Sorry about that. The `data-science-threat-intelligence` is another story.

**Review by dit7ya — APPROVED** [review]

**Pull request merged**